### PR TITLE
Aayog 0

### PR DESCRIPTION
**Pull request by koiralaa**

### CODE REVIEW

File: my-portfolio/src/p5_sketches/ball.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    xSpeed = 1;
+    ySpeed = 1;
+}
+var x, y, xSpeed, ySpeed;
+
+function draw(){
+    background(0);
+    ellipse(x, y, 16,16);
+    x+=xSpeed;
+    y+=ySpeed;
```
Comment (bises) [inline my-portfolio/src/p5_sketches/ball.js]: Is this your js file?

File: my-portfolio/package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "@angular/platform-browser": "~8.0.0",
     "@angular/platform-browser-dynamic": "~8.0.0",
     "@angular/router": "~8.0.0",
+    "p5": "^0.8.0",
```
Comment (bises) [inline my-portfolio/package.json]: What does this p5 library do??

File: my-portfolio/src/app/app.component.html [hunk 3]
```diff
@@ -1,20 +1,10 @@
-<!--The content below is only a placeholder and can be replaced.-->
+<head>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.8.0/p5.js"></script>
```
Comment (bises) [inline my-portfolio/src/app/app.component.html]: What is this used for?

File: my-portfolio/src/app/app.component.spec.ts [hunk 4]
```diff
@@ -1,9 +1,13 @@
 import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule
```
Comment (bises) [inline my-portfolio/src/app/app.component.spec.ts]: Why do you need this?

**Comment by bises** [comment]
Seems like there is a merge conflict also

File: my-portfolio/src/app/app.component.spec.ts (hunk 4 above)
Comment (koiralaa) [reply my-portfolio/src/app/app.component.spec.ts]: Ng new app garda it automatically added.

File: my-portfolio/src/app/app.component.html (hunk 3 above)
Comment (koiralaa) [reply my-portfolio/src/app/app.component.html]: This is the p5 library, that has the codes for sketches.

File: my-portfolio/package.json (hunk 2 above)
Comment (koiralaa) [reply my-portfolio/package.json]: It's from the processing org website, I use that to make all those codes that I want to add to my website.

File: my-portfolio/src/p5_sketches/ball.js (hunk 1 above)
Comment (koiralaa) [reply my-portfolio/src/p5_sketches/ball.js]: yes, this is supposed to display the canvas and do the animation, but isn't working.

**Pull request merged**